python/lsst/ip/diffim/makeKernel.py

```diff
@@ -58,6 +58,17 @@ class MakeKernelConfig(PsfMatchConfig):
         target=SingleFrameMeasurementTask,
         doc="Initial measurements used to feed stars to kernel fitting",
     )
+    fwhmExposureGrid = lsst.pex.config.Field(
+        doc="Grid size to compute the average FWHM in an exposure",
+        dtype=int,
+        default=10,
+    )
+    fwhmExposureBuffer = lsst.pex.config.Field(
+        doc="Fractional buffer margin to be left out of all sides of the image during construction"
+            "of grid to compute average FWHM in an exposure",
+        dtype=float,
+        default=0.05,
+    )
```

Comment:
Are these fieldnames helpful? They are not immediately apparent. We should at least have 'PSF' in their names.

Reply:
I think it is sufficiently clear if you expand the docstring to be `...average PSF FWHM...`